CommonWallet/Core/Classes/Modules/Transfer/ViewModel/TransferViewModelFactory.swift

```diff
@@ -103,7 +103,7 @@ extension TransferViewModelFactory: TransferViewModelFactoryProtocol {
         let feeDisplaySettings = feeDisplaySettingsFactory
             .createFeeSettingsForId(fee.feeDescription.identifier)
 
-        let amountFormatter = amountFormatterFactory.createTokenFormatter(for: feeAsset)
+        let amountFormatter = amountFormatterFactory.createFeeTokenFormatter(for: feeAsset)
```

Comment:
I think we also display fee on confirmation screen and tx details screen. Please, check corresponding factories too